network/dag/notifier.go

```diff
@@ -327,6 +300,43 @@ func (p *notifier) retry(event Event) {
 	}(p.ctx)
 }
 
+// runNow is used to call the receiverFn synchronously.
+// This is used for the first run.
+func (p *notifier) runNow(event Event) error {
```

Comment:
should be named `notifyNow`?

Reply:
some leftover terms from when it was a 'job' runner.